src/player/lib/audio-manager/HowlerAudioManager.ts

```diff
@@ -20,6 +22,12 @@ export class HowlerAudioManager implements IAudioManager {
     makeAutoObservable(this);
   }
 
+  loop(repeat: boolean) {
+    this.isLooping = !this.isLooping;
+
+    this.howl?.loop(repeat);
+  }
+
```

Comment:
Sorry, I haven't see this code.

We run the risk to be desynchronized between our HowlerAudioManager and Howl here.

We should either

```ts
loop(repeat: boolean) {
    this.isLooping = repeat;

    this.howl?.loop(repeat);
  }
```

or 

```ts
loop() {
    this.isLooping = !this.isLooping;

    this.howl?.loop(!this.isLooping);
}
```

See? Both our AudioManager and Howler will be updated with the **same state** no matter what the **initial state** is.

Reply:
Yes, good point, thank you! 

Corrected.